extensions/amp-timeago/0.1/validator-amp-timeago.protoascii

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-timeago
+  html_format: AMP
+  tag_name: "SCRIPT"
+  satisfies: "amp-timeago extension .js script"
+  requires: "amp-timeago"
+  extension_spec: {
+    name: "amp-timeago"
+    allowed_versions: "0.1"
+    allowed_versions: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-timeago>
+  html_format: AMP
+  tag_name: "AMP-TIMEAGO"
+  satisfies: "amp-timeago"
+  requires: "amp-timeago extension .js script"
+  attrs: {
```

Comment:
@edhollinghurst Based on @jridgewell's suggestion, let's restrict the attribute to ISO date format. 

I found the following regex on stackoverflow and tested it a bit, seems to work well allowing variants like omitting milliseconds, timezone
`(\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d:[0-5]\d\.\d+)|(\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d:[0-5]\d)|(\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d)` @jridgewell @honeybadgerdontcare in case you know of a better regex for ISO 

so we would need to do:
```
{
name: "datetime"
mandatory: true
value_regex: "(\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d:[0-5]\d\.\d+)|(\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d:[0-5]\d)|(\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d)"
}
```